### PR TITLE
Update electron to 11.1.1

### PR DESCRIPTION
**Pull request by Hinton**
Update electron to 11.1.1.

Relevant electron deprecation:

 * process.windowStore is now read-only, which required some minor refactoring in `src/electron/utils.ts`.
 * `app.getName()` is deprecated and has been replaced by `app.name`.

### CODE REVIEW

File: src/electron/utils.ts [hunk 1]
```diff
@@ -16,11 +16,12 @@ export function isMacAppStore() {
...
     const isWindows = process.platform === 'win32';
-    if (isWindows && !process.windowsStore &&
+    let windowsStore = process.windowsStore;
+    if (isWindows && !windowsStore &&
         process.resourcesPath.indexOf('8bitSolutionsLLC.bitwardendesktop_') > -1) {
-        process.windowsStore = true;
+        windowsStore = true;
     }
-    return isWindows && process.windowsStore && process.windowsStore === true;
+    return isWindows && windowsStore && windowsStore === true;
```
Comment (cscharf) [inline src/electron/utils.ts:24]: Can't this be simplified down to `return isWindows && windowsStore === true;` ?

**Review by cscharf — COMMENTED** [review]
Looks good, one boolean logic nit, but otherwise will hold until after Jan release.

File: src/electron/utils.ts (hunk 1 above)
Comment (flying-sausages) [reply src/electron/utils.ts:24]: is this nitpick the last thing holding off native possible Apple Silicon support?

Comment (cscharf) [reply src/electron/utils.ts:24]: Hi @flying-sausages, no, this is not holding anything up, we just were waiting for our prior release before approving/merging this in. This nit was just general feedback.

**Review by cscharf — DISMISSED** [review]

File: src/electron/utils.ts (hunk 1 above)
Comment (flying-sausages) [reply src/electron/utils.ts:24]: Sorry, my phrasing wasn't the most appropriate! Thanks for the clarification :)

Comment (Hinton) [reply src/electron/utils.ts:24]: @cscharf Probably, can't see any case where this simplification would not work.

**Review by cscharf — APPROVED** [review]

**Pull request merged**